src/base/net/downloadmanager.cpp

```diff
@@ -75,6 +75,9 @@ namespace
             Preferences::instance()->setNetworkCookies(cookies);
         }
 
+        using QNetworkCookieJar::allCookies;
+        using QNetworkCookieJar::setAllCookies;
```

Comment:
Just use protected base names as public here, instead of overriding these methods.